.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -3,7 +3,13 @@ name: Bug report
 about: Create a report to help us improve
 ---
 <!-- Please search existing issues to avoid creating duplicates. -->
+
+<!--
+For the EdgeDB Version: run `edgedb` from your project directory copying the first line of output or query `select sys::get_version_as_str();`
```

Comment:
You can get the version as a oneliner: `edgedb query 'select sys::get_version_as_str()'`

Reply:
Thanks, @elprans. I circled back and updated this across the new and updated issue templates.